CONTRIBUTING.md

```diff
@@ -303,11 +301,11 @@ GitHub has a [nice introduction][github-flow] to the pull request workflow, but
 
 ## The process of writing chapters
 
-- Fork the repository from the alan turing version if you have not done so already.
-- On the alan turing version create a branch with the same name as the chapter to be written.
-- On your fork create a branch with the same name and create a markdown file on it.
+- Fork the repository from [`alan-turing-institute/the-turing-way`](https://github.com/alan-turing-institute/the-turing-way) if you have not done so already.
+  - On your fork create a branch with the name as the chapter to be written and create a markdown file on it.
+- If you don't want to create a fork (or a local copy), you can also create a branch on the main repository with the same name as the chapter to be written.
```

Comment:
Should it be added that this can only be done if you're a contributor on the project (otherwise GitHub automatically makes a fork for you).

Reply:
That's true. Yes. That should be edited into that.

Reply:
Added the details.